model/gcp/registry.yaml

```
@@ -39,6 +39,75 @@ groups:
           [`CLOUD_RUN_TASK_INDEX`](https://cloud.google.com/run/docs/container-contract#jobs-env-vars)
           environment variable.
         examples: [0, 1]
+  - id: registry.gcp.apphub
+    type: attribute_group
+    display_name: GCP - AppHub Attributes
+    brief: >
+      This document defines attributes AppHub will apply to resources in GCP.
+      See [AppHub overview](https://cloud.google.com/app-hub/docs/overview).
+    attributes:
+      - id: gcp.apphub.application.container
+        type: string
+        stability: development
+        brief: >
+          The container within GCP where the AppHub application is defined.
+        examples: ['projects/my-container-project']
+      - id: gcp.apphub.application.location
+        type: string
+        stability: development
+        brief: >
+          The GCP zone or region where the application is defined.
+        examples: ['us-central1']
+      - id: gcp.apphub.application.id
+        type: string
+        stability: development
+        brief: >
+          The name of the application as configured in AppHub.
+        examples: ['my-application']
+      - id: gcp.apphub.service.id
+        type: string
+        stability: development
+        brief: >
+          The name of the service as configured in AppHub.
+        examples: ['my-service']
+      - id: gcp.apphub.service.environment_type
+        type: string
+        stability: development
+        brief: >
+          Environment of an application, service, or workload is the stage of a software lifecycle, for example, production, staging, or development.
+        note: >
+          [See AppHub properties and attributes](https://cloud.google.com/app-hub/docs/overview#properties-attributes)
+        examples: ['DEVELOPMENT', 'TEST', 'STAGING', 'PRODUCTION']
+      - id: gcp.apphub.service.criticality_type
```

would 
```suggestion
      - id: gcp.apphub.service.criticality.type
```

work too?

---

see other comment